src/resolver.js

```diff
@@ -36,11 +40,11 @@ export default function resolve({http, fetch, spec, url, baseDoc, mode, allowMet
   }
   else {
     // Store the spec into the url provided, to cache it
-    plugins.refs.docCache[baseDoc] = spec
+    docCache[baseDoc] = spec
```

Comment:
It seems like something like this could be done inside mapSpec. That ways we don't have to leak docCache to JS.